### ISSUE TITLE
Revise component subscriptions

### ISSUE THREAD

**Issue by FilipLeitner** [hslayers/hslayers-ng#4666]
Shared hslayers fucntionality should not rely on subscriptions created in components.

Example:
An app should be able to consume functionality thats provided by layermanager service such as event triggers for newly added layers
<img width="312" alt="image" src="https://github.com/hslayers/hslayers-ng/assets/44566616/c7078292-3d6e-40ad-a352-7129ba53cb19">
However this is currently not possible without layermanager component as it is its constructor which creates a handler.
<img width="291" alt="image" src="https://github.com/hslayers/hslayers-ng/assets/44566616/6b88f57b-3644-4ac3-96b1-b43053c867d9">

In other words subscriptions or other actions that concern the functionality of services should be initiated in services (either  through helpers or directly ) not panel components

**Comment by FilipLeitner** [comment]
Went through search results for regex `ce\.[\w\d]{1,50} =` (ce\. as ending of word service followed word character (letters, digits, or underscores) or digit, between 1 and 20 times)  in `*.component.ts` files and found nothing suspicious so it seem this was an exception.
